projects/1.18/assets/industrial-foregoing/industrialforegoing/lang/zh_cn.json

```
@@ -479,6 +479,28 @@
   "tooltip.titanium.facing_handler.blazeinput": "输入烈焰棒",
   "tooltip.titanium.facing_handler.brewinginput": "输入酿造",
   "tooltip.titanium.facing_handler.charging": "充能中",
+  "tooltip.titanium.facing_handler.furnace.input_0": "输入",
+  "tooltip.titanium.facing_handler.slimey.input_0": "输入牛奶",
+  "tooltip.titanium.facing_handler.slimey.input_1": "输入粘液球",
+  "tooltip.titanium.facing_handler.culinary.input_0": "输入食物",
+  "tooltip.titanium.facing_handler.potion.input_0": "输入药水",
+  "tooltip.titanium.facing_handler.disenchantment.input_0": "输入附魔",
```

```suggestion
  "tooltip.titanium.facing_handler.disenchantment.input_0": "附魔物品输入",
```
这一串翻译成 XX 输入/输入端 是否更好？看起来这是配置机器侧面功能的语句，Input 应为名词

---

ping @ArgonCrystal